client/pages/Login/utils/LoginValidator.tsx

```diff
@@ -13,8 +14,17 @@ const validatePassword = (password: string): string => {
   return null
 }
 
+const validatePasswords = (password: string, password2: string): string => {
+  console.log(password, password2)
+  if (password !== password2) return 'login.noMatchPasswords'
+  return null
+}
+
 export const LoginValidator = {
   localValidate: (email: string, password: string) => {
     return { email: validateEmail(email), password: validatePassword(password) }
   },
+  invitationValidate: (email: string, password: string, password2: string) => {
+    return { ...LoginValidator.localValidate(email, password), password2: validatePassword(password2), passwords: validatePasswords(password, password2) }
+  },
```

Comment:
For password2 it's enough to check if it matches password1, password1 has to be valid